spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -999,6 +1000,19 @@ def send_app_submission(app_id, version, data)
 
       handle_itc_response(r.body)
 
+      messages = r.body.fetch("messages", {})
+      errors = messages.fetch('error', nil)
+
+      if !errors.nil? and !errors.empty? and errors.first.include?("Problem processing review submission.")
```

Comment:
What other reasons could there be for that besides "Waiting for review"? Isn't it kind of dangerous to just reject the app even if the error doesn't directly

Reply:
@janpio This error message is returned when the app is already submitted for review. The status could be "Waiting for review", "In Review", "Pending developer release". In all three cases it will reject the app and submit again.

Reply:
I think we should change the param name from `reject_if_waiting_for_review` to something more generic like `reject_if_already_submitted` or `reject_if_previously_submitted`. I am open to suggestions if you have any other name. We will have to change the description of the param accordingly.

Reply:
I agree with @janpio that assuming that if the error received matches that string the error is due to the review process is already going on. The response does not include any other error that ensures that the error is produced by the application being already sent to review?

Reply:
Is there other solution for this problem? Is there a better way to check if the app is only in the above mentioned states?